packages/components/avatar/src/avatar.styles.ts

```diff
@@ -3,23 +3,94 @@ import { AvatarProps, AvatarStyledProps } from './avatar.types';
 
 const handleSize = (size: AvatarProps['size']) => {
   switch (size) {
+    case 'xxs':
+      return 'font-size: 0.75rem; width: 1.5rem; height: 1.5rem;';
+    case 'xs':
+      return 'font-size: 0.875rem; width: 2rem; height: 2rem;';
     case 'sm':
-      return 'font-size: 0.75rem; width: 2rem; height: 2rem;';
+      return 'font-size: 1rem; width: 2.5rem; height: 2.5rem;';
     case 'md':
-      return 'font-size: 0.875rem; width: 2.5rem; height: 2.5rem;';
+      return 'font-size: 1.25rem; width: 3rem; height: 3rem;';
     case 'lg':
     default:
-      return 'font-size: 1rem; width: 3rem; height: 3rem;';
+      return 'font-size: 1.5rem; width: 4rem; height: 4rem;';
     case 'xl':
-      return 'font-size: 1.25rem; width: 4rem; height: 4rem;';
+      return 'font-size: 2rem; width: 6rem; height: 6rem;';
+    case 'xxl':
+      return 'font-size: 3rem; width: 10rem; height: 10rem;';
+  }
+};
+const handleMargin = (size: AvatarProps['size']) => {
+  switch (size) {
+    case 'xxs':
+      return '-0.50rem';
+    case 'xs':
+      return '-0.75rem';
+    case 'sm':
+      return '-1rem';
+    case 'md':
+      return '-1.5rem';
+    case 'lg':
+      return '-1.5rem';
+    case 'xl':
+      return '-2rem';
+    case 'xxl':
+      return '-3rem';
+    default:
+      return '-1.5rem';
+  }
+};
+
+const handleBoxShadow = (size: AvatarProps['size']) => {
+  const insetValue = 'inset 0 0 0 ';
+
+  switch (size) {
+    case 'xxs':
+    case 'xs':
+    case 'sm':
+      return `${insetValue}0.063rem`;
+    case 'md':
+    case 'lg':
+      return `${insetValue}0.125rem`;
+    case 'xl':
+      return `${insetValue}0.188rem`;
+    case 'xxl':
+      return `${insetValue}0.25rem`;
+    default:
+      return `${insetValue}0.125rem`;
   }
 };
 
+const handleBadge = (size: AvatarProps['size']) => {
+  switch (size) {
+    case 'xxs':
+      return 'top:-8%; right:-8%';
+    case 'xs':
+      return 'top:-6%; right:-6%';
+    case 'sm':
+      return 'top:5%; right:5%';
+    case 'md':
+      return 'top:5%; right:4%';
+    case 'lg':
+      return 'top:9%; right:9%';
+    case 'xl':
+      return 'top:13%; right:13%';
+    case 'xxl':
+      return 'top:15%; right:15%';
+    default:
+      return '-1.5rem';
+  }
+};
+
+export const AvatarWrapper = styled.div`
+  position: relative;
+`;
+
 export const AvatarStyled = styled.div<{
   styled: Required<AvatarStyledProps>;
```

Comment:
xxs, xs, sm, md, lg, xl, xxl: font-weight 400; 

can you change the border on badge to: box-shadow: inset 0 0 0 1px red;?

Reply:
This request is related to the badge not to the avatar.